mautic/mautic
# [UI] Accent colors

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This allows you to set your preferred color.

Also:
- Adds styles for popovers
- Adds popover about solarized themes
- Includes minor fixes for the account page
- Makes the JS function more reliable when no accent colors are previously set (they default)
- Removes support for secondary brand color
- Adds 25 predefined accents
- Creates Twig template for popover
- Creates Twig template for accents
- Several fixes to improve the platform compatibility with themes
- Nests remix icon pack classes into the main icon class to improve specificity and allow devs to customize icon size in buttons

![image](https://github.com/user-attachments/assets/582ace75-9dc4-4ee4-b280-9b1c582f912b)

![image](https://github.com/user-attachments/assets/9840a492-63b0-4be7-8001-23c2f69410bf)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Account
3. Open the Appearance tab
4. Play with the colors then check the entire platform
